data/tutorials/getting-started/02_fst_program.md

```diff
@@ -0,0 +1,308 @@
+---
+id: how-to-write-an-ocaml-program
+title: How to Write an OCaml Program
+description: >
+  Get something real out of the damm French thing!
+category: "getting-started"
+---
+
+[TOC]
+
+## Get Something Done
+
+This tutorial is the last part of the three-part series. Please ensure you have completed [“Installing OCaml”](linkhere) and [`“A Tour of OCaml”](linkhere) before proceeding onto your first OCaml project. You will need your environment been set up as described in the ["Installing OCaml"](Install_link).
+
+In this tutorial, we start working with files containing OCaml source code and compiling them to produce excutable binaries. However, this is not a detailed tutorial on OCaml compilation, project modularisation, or dependencies management; it only gives a glimpse at those topics. The goal is to sketch the bigger picture before extensively presenting topics in order to avoid getting lost in the details. 
+
+In other words, breadth-first learning instead of depth-first learning.
+
+## Compiling OCaml Programs
+
+OCaml comes with both an interpreter and a compiler. In the previous tutorial, only the interpreter was used. This section gives a glimpse of how to use the compiler. 
+
+We start by seting up a traditional “Hello World!” project using Dune, OCaml’s build system. The following creates it under the name `hello`:
+
+```shell
+$ dune init proj hello
+Entering directory '/home/cuihtlauac/caml/ocaml.org/hello'
+Success: initialized project component named hello
+```
+
+The project is stored in a directory named `hello`. The `tree` command lists the files and directories created. 
+
+```shell
+$ cd hello
+$ tree
+.
+├── bin
+│   ├── dune
+│   └── main.ml
+├── _build
+│   └── log
+├── dune-project
+├── hello.opam
+├── lib
+│   └── dune
+└── test
+    ├── dune
+    └── hello.ml
+
+4 directories, 8 files
+```
+
+OCaml source files have `.ml` extension, which stands for “Meta Language.” Meta Language (ML) is the ancestor of OCaml, this is also what the “ml” stands for in “OCaml.” Here is the content of the `bin/main.ml` file:
+
+```shell
+$ cat ./bin/main.ml
+let () = print_endline "Hello, World!"
+```
+
+The project-wide metadata is available in the `dune-project` file. Each folder containing source files that need to be built must contain a `dune` file explaining how.
+
+This builds the project:
+```shell
+$ dune build
+Entering directory '/home/cuihtlauac/caml/ocaml.org'
```

Comment:
```suggestion
Entering directory '/home/cuihtlauac/caml/ocaml.org'
```
Again, I don't get this. It appears that the computer does something behind the scenes (as the text in the CLI blinked), but then it just gives me another prompt ready for another command.